lib/mail_room/arbitration.rb

```diff
@@ -0,0 +1,14 @@
+module MailRoom
+  module Arbitration
+    def self.[](name)
+      require_relative("./arbitration/#{name}")
```

Comment:
I'm not sure how to do that cleanly without repeating the case/when statement. Either way, if the delivery/arbitration method doesn't exist, an error seems in place.